source/user-manual/configuring-cluster/index.rst

```diff
@@ -0,0 +1,192 @@
+    | :ref:`RPM packages <wazuh_server_rpm_centos_filebeat>`                 | Install Filebeat on CentOS.                                 |
+    +------------------------------------------------------------------------+-------------------------------------------------------------+
+    | :ref:`RPM packages <wazuh_server_rpm_fedora_filebeat>`                 | Install Filebeat on Fedora.                                 |
+    +------------------------------------------------------------------------+-------------------------------------------------------------+
+    | :ref:`RPM packages <wazuh_server_rpm_oracle_filebeat>`                 | Install Filebeat on Oracle Linux.                           |
+    +------------------------------------------------------------------------+-------------------------------------------------------------+
+    | :ref:`RPM packages <wazuh_server_rpm_rhel_filebeat>`                   | Install Filebeat on Red Hat Enterprise Linux.               |
+    +------------------------------------------------------------------------+-------------------------------------------------------------+
+    | :ref:`RPM packages <wazuh_server_rpm_suse_opensuse_filebeat>`          | Install Filebeat on CentOS/RHEL/Fedora.                     |
+    +------------------------------------------------------------------------+-------------------------------------------------------------+
+    | :ref:`DEB packages <wazuh_server_deb_filebeat>`                        | Install Filebeat on Debian/Ubuntu.                          |
+    +------------------------------------------------------------------------+-------------------------------------------------------------+
+
+    **Installing Splunk forwarder:**
+
+    +-------------------------------------------------------------------+-------------------------------------------------------------+
+    | Type                                                              | Description                                                 |
+    +===================================================================+=============================================================+
+    | :ref:`RPM/DEB packages <splunk_forwarder>`                        | Install Splunk forwarder for RPM or DEB based OS.           |
+    +-------------------------------------------------------------------+-------------------------------------------------------------+
+
+
+.. _deploy_wazuh_agents_cluster:
+
+.. topic:: Pointing agents to the cluster nodes
+
+    Finally, the configuration of the agents has to be modified in order to report to the cluster.
+    In this case the agent 001 will report to the worker01-node node, to achieve this it is necessary to modify the information contained in the labels ``<client><server>`` in the file ``/var/ossec/etc/ossec.conf``, inside we will place the IP of the node we want to report, in our case it would be like this:
```

Comment:
```suggestion
    In this case the agent 001 will report to the worker01-node node. To achieve this we must modify the information contained in the labels ``<client><server>`` in the file ``/var/ossec/etc/ossec.conf``, where we will place the IP of the node we want to report to, in our case it would look like this:
```